reducer-todo/src/components/Todo.js

```diff
@@ -0,0 +1,29 @@
+/* (Reminder: ({ dispatch }) is simply using JS destructuring assignment to extract dispatch from this.props object. dispatch() is the method used to dispatch actions and trigger state changes)
```

Comment:
Great job with the explanation!  Just small point.  `this` only exists inside class components.